pagopa/io-app
# fix: [PE-699] CGN detail header title on scroll

## Short description
Add missing header title on scroll

## How to test
- login into the app
- go to cgn card trough wallet (add it if necessary)
- scroll down and check if the title is visible in the sticky header

![image](https://github.com/user-attachments/assets/b5881f1b-4f30-4b02-a116-a5d74e52d44a)
